mtp_noms_ops/apps/security/templatetags/security.py

```diff
@@ -82,6 +83,22 @@ def format_resolution(resolution):
     return credit_resolutions.get(resolution, resolution)
 
 
+@register.filter
+def check_description(check):
+    description = check['description']
+    if isinstance(description, list):
+        description = '. '.join(description)
```

Comment:
I'm not going to block on this, but it looks like this is a comma in the wireframes

Reply:
the full list won't make sense without full stops, unfortunately. each description fragment is generated as a standalone sentence